.dockerignore

```diff
@@ -0,0 +1,2 @@
+/node_modules/
+.DS_Store
```

Comment:
Should include `/dist/` as well. Generally speaking, we keep all entries from `.gitignore` in this file as well.